### ISSUE TITLE
bug(profileParamsToPopulate): role does not populate - 1.5.0-rc1

### ISSUE THREAD

**Issue by darrenfurr** [prescottprue/react-redux-firebase#203]
**Do you want to request a *feature* or report a *bug*?**  
bug

**What is the current behavior?**
When using the role recipe with 1.5.0-rc1, the profile does not populate.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net or similar.**
Repeat the steps in the Role recipe outlined here:
http://react-redux-firebase.com/docs/recipes/roles.html

config.js contains:
```
profileParamsToPopulate: [
      { child: 'role', root: 'roles' }, // populates user's role with matching role object from roles
  ]
```
component contains:
```
@connect(
  ({ firebase }) => ({
    auth: pathToJS(firebase, 'auth'),
    account: pathToJS(firebase, 'profile'),
  })
)
```
inside the component just `console.log(account)` & you'll see the role is `undefined`.

**What is the expected behavior?**
role property is populated

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**
Works in 1.4.4

**Comment by prescottprue** [comment]
@darrenfurr Thanks for making an issue for this. Looking into it some more today.

**Comment by prescottprue** [comment]
@darrenfurr I was not able to replicate this in the material example. Going to reach out over Gitter to debug a bit more with you, the post results back here.

**Comment by prescottprue** [comment]
[`v1.5.0-rc.2` was released](https://github.com/prescottprue/react-redux-firebase/releases/tag/v1.5.0-rc.2) with a fix for profile populating. The material example now has the profileParamsToPopulate config included.

**Comment by darrenfurr** [comment]
Confirmed. This is fixed with 1.5.0-rc.2
`Object {displayName: "Darren Furr", email: "xxxxxxxx@gmail.com", providerData: Array(1), role: Object}`

**Comment by prescottprue** [comment]
@darrenfurr Great to know. Its seems that `v1.5.0` is stable 😄
